### PR TITLE
test(ui): add e2e regression test for issue #15239

### PR DESCRIPTION
**Pull request by hoorayimhelping**
Connects #15239

Adds e2e regression test for above ticket ☝️ 

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: ui/cypress/e2e/dashboardsView.test.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ describe('Dashboard', () => {
     })
   })
 
-  it('can edit a dashboards name', () => {
+  it("can edit a dashboard's name", () => {
```
Comment (Asalem1) [inline ui/cypress/e2e/dashboardsView.test.ts]: most excellent

Comment (desa) [inline ui/cypress/e2e/dashboardsView.test.ts]: double quotes on purpose?

File: ui/cypress/e2e/dashboardsView.test.ts [hunk 2]
```diff
@@ -60,6 +60,39 @@ describe('Dashboard', () => {
...
+
+    // Add an empty celly cell
+    cy.getByTestID('add-cell--button').click()
+    cy.getByTestID('save-cell--button').click()
+    cy.getByTestID('cell--view-empty').should('be.visible')
+
+    let cellContent
+    cy.getByTestID('cell--view-empty')
+      .invoke('text')
+      .then(cellContent => {
```
Comment (Asalem1) [inline ui/cypress/e2e/dashboardsView.test.ts]: @hoorayimhelping , why're we calling the parameter that's being passed in the `then` the same thing as the variable that's being instantiated on line 78:

https://github.com/influxdata/influxdb/pull/15590/files#diff-33b992b7a030a90957e222522b90a778R78

File: ui/cypress/e2e/dashboardsView.test.ts (hunk 1 above)
Comment (hoorayimhelping) [reply ui/cypress/e2e/dashboardsView.test.ts]: it used to be

>can edit a dashboards name

so i added the possessive apostrophe and the double quote so we wouldn't have to escape the apostrophe

File: ui/cypress/e2e/dashboardsView.test.ts (hunk 2 above)
Comment (hoorayimhelping) [reply ui/cypress/e2e/dashboardsView.test.ts]: oops, didn't mean to leave `let cellContent` in there. i bet lint will complain now. good catch!

**Pull request merged**